docs/intro.rst

```diff
@@ -640,52 +679,61 @@ Configuration variables
 .. automodule:: pywbem.config
       :members:
 
+
 .. _`WBEM Servers`:
 
 WBEM servers
 ------------
 
-Pywbem supports communication with any WBEM server that conforms to the
-DMTF specifications. See section  :ref:`Standards conformance`.
-
-Server Specific Features
+Server-specific features
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-There are some specific non-specification features that are included in
-pywbem for support of WBEM server specific features including:
+Pywbem supports the following features of some specific WBEM servers that are
+additions to the DMTF standards:
+
+1. `OpenPegasus <https://collaboration.opengroup.org/pegasus/>`_
+
+   - Pywbem supports the Interop namespace ``root/PG_InterOp`` that is specific
+     to OpenPegasus. OpenPegasus also supports the standard Interop namespaces
+     (``interop``, ``root/interop``) but for backwards compatibility with
+     earlier versions of OpenPegasus, pywbem supports this old Interop
+     namespace, for example in its Interop namespace determination whose result
+     is exposed in the :attr:`pywbem.WBEMServer.interop_ns` property.
 
-1. OpenWBEM server - includes support for an authentication extension
-(OWLocal password-less local authorization) that is part of the OpenWBEM server.
+   - Pywbem supports the upper-case variant ``EMBEDDEDOBJECT`` of the respective
+     CIM-XML attribute that is specific to OpenPegasus, in addition to the
```

Comment:
change to "some releases of OpenPegasus"

Reply:
DONE.